sdk/search/search-documents/test/README.md

```diff
@@ -0,0 +1,20 @@
+# Testing
+
+To test this project, make sure to first build it properly by following our [building instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#building). Once the project is correctly built, you will be able to run the tests by following the [testing instructions](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md#testing).
+
+The Azure Cognitive Search client does not have any recorded tests and so, all the tests require an Azure Cognitive Search account to be set up beforehand. You can use existing Azure resources for the live tests, or generate new ones by using our [New-TestResources.ps1](https://github.com/Azure/azure-sdk-for-js/blob/master/eng/common/TestResources/New-TestResources.ps1) script, which will use an [ARM template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/search/test-resources.json) that already has all of the the necessary configurations.
+
+The Azure resources that are used by the tests in this project are:
+
+- An [Azure Cognitive Search](https://docs.microsoft.com/en-us/azure/search/search-what-is-azure-search) account, configured with the default parameters.
+
+You will also need to set the below environment variables to run the live tests:
+
+- `TEST_MODE`: Should have `live` assigned.
+- `SEARCH_API_ADMIN_KEY`: The primary key of your Azure Search account.
+- `SEARCH_API_ADMIN_KEY_ALT` (optional): The secondary key of your Azure Search account.
+- `ENDPOINT`: The endpoint of your Azure Search account.
```

Comment:
Just some thoughts unrelated to this PR..
Should we prefix the environment variables with a shorthand notation of the service-name?

Reply:
@ramya-rao-a I think your directions here are appreciated.

Reply:
I agree we should prefix resource variable with service-name. For example, it would be better if I want to run tests of two libraries without having to worry about potential name clashes.

Reply:
Seems like this should be worked on outside of this PR. Perhaps we should ask @xirzec

Reply:
I was hoping have this fixed as part of #8299 8463